lib/indexers/enforcements/index.js

```diff
@@ -0,0 +1,122 @@
+const { pick } = require('lodash');
+const deleteIndex = require('../utils/delete-index');
+
+const indexName = 'enforcements';
+const columnsToIndex = ['caseNumber', 'createdAt', 'updatedAt'];
+
+const indexEnforcement = (esClient, enforcement) => {
+  return esClient.index({
+    index: indexName,
+    id: enforcement.id,
+    body: {
+      ...pick(enforcement, columnsToIndex),
+      subjects: enforcement.subjects.map(subject => {
+        return {
+          ...pick(subject.profile, 'id', 'firstName', 'lastName'),
+          name: `${subject.profile.firstName} ${subject.profile.lastName}`,
+          establishment: subject.establishment.name,
```

Comment:
In an establishment search we also search the `keywords` property of establishments to support common aliases (e.g. RVC, UCL). We may want to do the same here.